marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/keystore/AWSKeyStoreInit.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2012-2020 MarkLogic Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.marklogic.hub.central.keystore;
+
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder;
+import com.amazonaws.services.simplesystemsmanagement.model.GetParameterRequest;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.io.InputStream;
+
+@Component
+@Profile("aws")
+public class AWSKeyStoreInit extends AbstractKeyStoreInit {
+
+    @Value("${aws.s3.bucket.name}")
```

Comment:
Just use "bucketName" and "keyName" unless we're certain that we'll have other properties on objects named "bucket" and "key".

Reply:
Used the same naming convention that we used for Azure i.e `cloud-platform.cloud-service.property-name`